### DISCUSSION TITLE
How show device count sensor in HA?

### DISCUSSION THREAD

**Discussion by cllatser** [ESPresense/ESPresense#422] [Support]
Good morning, how do I make the device counter sensor appear per room? Thank you very much.
![ESPresence](https://user-images.githubusercontent.com/65849393/159703818-4301eadc-aa70-412e-975b-447c9f1fc0e8.jpg)

**Comment by DTTerastar** [comment]
Putting any value in the count ids field and rebooting should make it appear.

**Reply by cllatser** [reply]
Okay. Thank you very much.
